test/tests/functional/pbs_apbasil_inventory.py

```diff
@@ -45,32 +45,37 @@
 @tags('cray', 'mom')
 class TestBasilInventory(TestFunctional):
     """
-    This test suite is for testing the support for BASIL 1.4 inventory
+    This test suite is for testing the support for BASIL 1.7/1.4 basil
```

Comment:
Please change the class name to something like TestBasilQuery if it also tests system query